test/addons/olm/start

```diff
@@ -21,21 +19,17 @@ def deploy(cluster):
     #   The CustomResourceDefinition "clusterserviceversions.operators.coreos.com"
     #   is invalid: metadata.annotations: Too long: must have at most 262144 bytes
     # See https://medium.com/pareture/kubectl-install-crd-failed-annotations-too-long-2ebc91b40c7d
-    kubectl.apply(
-        f"--filename={BASE_URL}/crds.yaml",
-        "--server-side=true",
-        context=cluster,
-    )
+    kubectl.apply("--kustomize", "crds", "--server-side=true", context=cluster)
 
     print("Waiting until cdrs are established")
     kubectl.wait(
         "--for=condition=established",
-        f"--filename={BASE_URL}/crds.yaml",
+        "--filename=https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.27.0/crds.yaml",
```

Comment:
This is not great, but the next commit eliminates this. Trying to do this in a better way is not worth the time.

Reply:
Yeah I felt this, I tried but did not find a way to pass kustomize file to kubectl wait command..
So I just kept it like this, because anyway I was going to improve this in next commit.